cdk/lib/__snapshots__/prism-ec2-app.test.ts.snap

```diff
@@ -693,6 +680,20 @@ dpkg -i /prism/prism.deb",
             "Key": "deletion_protection.enabled",
             "Value": "true",
           },
+          Object {
+            "Key": "access_logs.s3.enabled",
+            "Value": "true",
+          },
+          Object {
+            "Key": "access_logs.s3.bucket",
+            "Value": Object {
+              "Ref": "AccessLoggingBucket",
+            },
+          },
+          Object {
+            "Key": "access_logs.s3.prefix",
+            "Value": "",
```

Comment:
Probably worth adding some kind of prefix (maybe just `prism`?) as I think this bucket is intended to be shared by multiple services